Problems/10871/10871.py

```diff
@@ -0,0 +1,25 @@
+"""
+Name: Bitnuri Kim
+Problem: 10871
+
+https://www.acmicpc.net/problem/10871
+
+첫째 줄에 N과 X가 주어진다. (1 ≤ N, X ≤ 10,000)
+둘째 줄에 수열 A를 이루는 정수 N개가 주어진다. 주어지는 정수는 모두 1보다 크거나 같고, 10,000보다 작거나 같은 정수이다.
+
+X보다 작은 수를 입력받은 순서대로 공백으로 구분해 출력한다. X보다 작은 수는 적어도 하나 존재한다.
+"""
+
+
+N, X = map(int, input("N개 X정수 입력").split(' '))
+A = list(map(int, input("수열 N개 만큼 입력").split(' ')))
+B = []
+
+idx=0
+for i in A:
+    if i < X:
+        B.append(i)
+    idx = idx +1
+
+strB = " ".join(map(str,B)) #join은 str type만 가능
+print(strB)
```

Comment:
```suggestion
for i in A:
    if i < X:
        print(i, end = ' ')
```
print(변수, end =' ') 이렇게 사용을 하면 꼭 str과 join이 없어도 처리가 가능할 거 같아.
print 함수 내에 end = 를 사용하면, 끝에 어떻게 처리할지 결정할 수 있어서 지금 로직에선 활용 가능할 거 같넹ㅎㅎㅎ